test-app/overlays/dev/cms-content-cm.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cms-content-cm
+  annotations:
+    reloader.stakater.com/match: "true"
+data:
+  ELECTRONICS_THEME_URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-default-2.1.11.zip
+  ELECTRONICS_THEME_LOCATION: /mnt/cms-content/Themes/Electronics/default
+  B2B_THEME_URL: https://vc3prerelease.blob.core.windows.net/packages/vc-demo-theme-b2b-1.0.0-alpha.1171.zip
```

Comment:
Use for test app latest b2b release from vc-theme-b2b repository https://github.com/VirtoCommerce/vc-theme-b2b/releases/tag/3.0.7